app/retail/views.py

```diff
@@ -1577,3 +1578,51 @@ def styleguide_components(request):
     else:
         context = {}
         return TemplateResponse(request, 'styleguide_components.html', context)
+
+
+@require_http_methods(["GET",])
+def jtbd_earn(request):
+    data = JSONStore.objects.filter(view='jtbd', key='earn').first().data
+    context = {
+        'title': _('Earn | Gitcoin'),
+        'card_title': _('Gitcoin - Support public goods, support open source'),
+        'card_desc': _('Break free from your 9–5 to work on open source projects that matter')
+    }
+    context.update(data)
+    return TemplateResponse(request, 'jtbd/earn.html', context)
+
+
+@require_http_methods(["GET",])
+def jtbd_learn(request):
+    data = JSONStore.objects.filter(view='jtbd', key='learn').first().data
+    context = {
+        'title': _('Learn | Gitcoin'),
+        'card_title': _('Gitcoin - Support public goods, support open source'),
+        'card_desc': _('Learn how to build the decentralized web')
+    }
+    context.update(data)
+    return TemplateResponse(request, 'jtbd/learn.html', context)
+
+
+@require_http_methods(["GET",])
+def jtbd_connect(request):
+    data = JSONStore.objects.filter(view='jtbd', key='connect').first().data
+    context = {
+        'title': _('Connect | Gitcoin'),
+        'card_title': _('Gitcoin - Support public goods, support open source'),
+        'card_desc': _('Connect And Build With Top Open Source Developers')
+    }
+    context.update(data)
+    return TemplateResponse(request, 'jtbd/connect.html', context)
+
+
+@require_http_methods(["GET",])
+def jtbd_fund(request):
+    data = JSONStore.objects.filter(view='jtbd', key='fund').first().data
+    context = {
+        'title': _('Fund | Gitcoin'),
+        'card_title': _('Gitcoin - Support public goods, support open source'),
+        'card_desc': _('There’s No Better Way To Fund Open Source Projects')
+    }
+    context.update(data)
+    return TemplateResponse(request, 'jtbd/fund.html', context)
```

Comment:
^ @chibie Is it worth DRY-ing those code ?

Reply:
you think to create just a function with a condition? or how can be dry? 
Im ok on having it on their own def but open to suggestions

Reply:
> you think to create just a function with a condition

Yup ! exactly cause only the 
`data`/`title` / `*.html` are different.
If it's an overkill -> then let's leave it like this